README.md

```diff
@@ -109,3 +113,6 @@ npx eslint --fix .
 * Follow [semantic commit messages](https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716) when writing your commits
 * Branch names should be in the format `jdoe/name-of-feature` where `jdoe` is in the format first initial, last name. Use dashes (`-`) to separate words
 
+## updates
+Feb 20, 2023:
+  * Implemented the components into the Dasboard page.
```

Comment:
We don't need updates for now. The git commit history should have these changes already

Reply:
I have removed Updates section from README.md